src/sentry/utils/redis.py

```diff
@@ -266,6 +266,43 @@ def call_script(client, keys, args):
         """.format(
             path
         )
-        return script(keys, args, client)
+        if not script:
+            script.append(
+                Script(client, resource_string("sentry", posixpath.join("scripts", path)))
```

Comment:
Re: https://github.com/getsentry/sentry/pull/19678#discussion_r449311713 (can't reply there since the commit was force pushed over and no longer in the chain of commits here): As long as a client is explicitly passed, this doesn't get used, right? This is probably OK, setting it back to `None` might be a bit more conservative in case something changes unexpectedly, but either way works.

Reply:
I thought I replied to this but not seeing it. Yeah, passing an explicit client when calling the script will always override the value passed in `__init__`.